tsconfig.json

```diff
@@ -1,5 +1,7 @@
 {
   "compilerOptions": {
+    "declaration": true,
+    "declarationDir": "./dist",
```

Comment:
One consideration here, as a side note.

This will cause the typings to be generated each time `tsc` is being used.

You could use the configuration I used in #46 to only generate the typings when using the `rollup` command